oculus_interfaces/package.xml

```diff
@@ -2,7 +2,7 @@
 <package format="3">
     <name>oculus_interfaces</name>
     <version>0.0.1</version>
-    <description>TODO: Package description</description>
+    <description>TODO(hugoyvrn): Package description</description>
     <maintainer email="auguste@forssea-robotics.fr">Auguste BOURGOIS</maintainer>
     <license>BSD3</license>
```

Comment:
This code is open sourced, correct?

Reply:
Yes, I think so

Reply:
Is BSD3  license compatible with forssea copyright ? I think it is.